core/src/main/java/io/questdb/cairo/O3CallbackJob.java

```diff
@@ -40,6 +40,7 @@ public static void runCallbackWithCol(O3CallbackTask task, long cursor, @NotNull
         final int columnIndex = task.getColumnIndex();
         final int columnType = task.getColumnType();
         final long mergedTimestampsAddr = task.getMergedTimestampsAddr();
+        final long mergedCount = task.getMergeCount();
```

Comment:
consistency, merge or merge(d) ? merged what?

Reply:
perhaps `timestampMergeIndexEntryCount` we should make task properties have consistent names

Reply:
I changed it to `mergeCount`. Merge is the operation in O3, merge count means the number of rows in the merge section, I believe it is descriptive